zfsbootmenu/lib/zfsbootmenu-core.sh

```diff
@@ -1673,6 +1673,16 @@ load_key() {
     return 0
   fi
 
+  # Run early load_key hooks, if they exist
+  if [ -d /libexec/load_key.d ]; then
+    tput clear
```

Comment:
```suggestion
```
The screen should not be cleared before hooks run; if a hook needs a clear screen, do it there.

Reply:
Got it

Reply:
I can't commit your suggestion as I did it by myself